filters/encrypt/testing.go

```diff
@@ -75,6 +75,8 @@ func TestHmacSha256(t *testing.T, data []byte, w wrapping.Wrapper, salt, info []
 // TestMapField defines a const for a field name used for testing TestTaggedMap
 const TestMapField = "foo"
 
+const TestPublicMapField = "public-foo"
```

Comment:
godoc

Also any reason to not to group in a const block...
```
const (
	// TestMapField defines a const for a field name used for testing TestTaggedMap
	TestMapField       = "foo"

	// TestPublicMapField ...
	TestPublicMapField = "public-foo"
)
```

Reply:
ty!